docs: edit command help for context commands

bhardesty opened the pull request:
Edited the command help for the "rhoas context" commands.

I'm not sure about the generate-config command. I tried to include everything that it's potentially supposed to generate, but I'm not sure that all of the functionality is actually in place yet. So we might need to scale back the help text accordingly.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation change
- [ ] Other (please specify)

wtrocki commented on pkg/core/localize/locales/en/cmd/context.en.toml:
```diff
@@ -3,28 +3,33 @@ one='Group, share and manage your rhoas services'
 [context.cmd.longDescription]
 one='''
-rhoas context commands allow developers to:
+Group your application service instances into service contexts.
```
As first sentence it sounds strange to me.
Maybe shorter form?

```suggestion
Group your service instances into contexts.
```

bhardesty replied:
In general for the CLI, I try to specify "application services" when referring to one of the RHOAS services. To distinguish them from the services that devs are building. Instead of "application services", how about "RHOAS services"?

wtrocki replied:
We cannot use acronyms like rhoas

wtrocki commented on pkg/core/localize/locales/en/cmd/context.en.toml:
```diff
@@ -3,28 +3,33 @@ one='Group, share and manage your rhoas services'
 
-Additionally developers can specify custom location for contexts by setting $RHOAS_CONTEXT environment variable in their profile.
-Setting $RHOAS_CONTEXT to "./rhoas.json" will load contexts from the current folder.
+You can also use the service context to automatically generate configuration files that you need to use those application service instances in other development platforms and tools. For example, the service context can generate the following types of configurations:
```
Pro tip. For very long lines we can use different toml syntaxes.
Simplest way is to break like with `\`

bhardesty replied:
:+1:

wtrocki commented on pkg/core/localize/locales/en/cmd/context.en.toml:
```diff
@@ -3,28 +3,33 @@ one='Group, share and manage your rhoas services'
+You can also use the service context to automatically generate configuration files that you need to use those application service instances in other development platforms and tools. For example, the service context can generate the following types of configurations:
+
+- Standard environment variables for use in local development and tooling
+- Java properties files for use in Quarkus, Apache Kafka, and so on
```
```suggestion
- Java properties files that can be used in Quarkus, Apache Kafka
```

bhardesty replied:
Done.

rkpattnaik780 commented on pkg/core/localize/locales/en/cmd/context.en.toml:
```diff
@@ -3,28 +3,33 @@ one='Group, share and manage your rhoas services'
-  * Manage different service contexts by switching, listing and removing service contexts 
-  * Share context with others to use the same set of services
-  * Generating configuration for connecting to the services from various platforms and tools
+A service context is a group of application service instances and their configuration details. By using service contexts, you can group together application service instances that you want to use together.
```
Since contexts hold only service identifiers, I don't think we should use the term configuration details here.

bhardesty replied:
Thanks! Done.

rkpattnaik780 reviewed (commented):
Thanks a lot for refining the docs. Looks great. Just a minor query.

wtrocki commented on pkg/core/localize/locales/en/cmd/context.en.toml:
```diff
@@ -177,11 +193,20 @@ one='Context deleted successfully'
+This command generates configuration files that you need to use application service instances in other development platforms and tools. For example, the service context can generate the following types of configurations:
+
+- Standard environment variables for use in local development and tooling
+- Java properties files for use in Quarkus, Apache Kafka, and so on
```
```suggestion
- Java properties files for use in Quarkus, Apache Kafka,
```

bhardesty replied:
Done.

wtrocki commented on pkg/core/localize/locales/en/cmd/context.en.toml:
```diff
@@ -177,11 +193,20 @@ one='Context deleted successfully'
+
+- Standard environment variables for use in local development and tooling
+- Java properties files for use in Quarkus, Apache Kafka, and so on
+- Service binding configuration and service connections for the RHOAS Operator
```
THis is not there. We can remove it

wtrocki commented on pkg/core/localize/locales/en/cmd/context.en.toml:
```diff
@@ -177,11 +193,20 @@ one='Context deleted successfully'
+- Standard environment variables for use in local development and tooling
+- Java properties files for use in Quarkus, Apache Kafka, and so on
+- Service binding configuration and service connections for the RHOAS Operator
+- Configuration for Helm Charts and Kubernetes
```
I think we need JSON format. We do not have helm charts support as for now

wtrocki commented:
@bhardesty We need to resolve conflict.

bhardesty commented:
Thanks for the review, @wtrocki and @rkpattnaik780. I addressed all of your comments and fixed the merge conflict. I think there's just one outstanding question re: would it be better to use "application services" or "RHOAS services"?
